c-dependencies/js-compute-runtime/host_call.h

```diff
@@ -1,56 +1,104 @@
 #ifndef JS_COMPUTE_RUNTIME_HOST_CALL_H
 #define JS_COMPUTE_RUNTIME_HOST_CALL_H
 
-#include "js-compute-builtins.h"
+// TODO: remove these once the warnings are fixed
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winvalid-offsetof"
+#include "jsapi.h"
+#pragma clang diagnostic pop
 
 #include "xqd.h"
 
+typedef enum class FastlyStatus {
```

Comment:
You don't need to typedef this definition.
```suggestion
enum class FastlyStatus {
```